recipe/meta.yaml.orig

```diff
@@ -0,0 +1,51 @@
+{% set name = "winpty" %}
+{% set version = "0.4.3" %}
```

Comment:
Guessing this file was added in error. Please remove by using [`git rm`]( https://git-scm.com/docs/git-rm ).